azure-toolkit-libs/azure-toolkit-containerapps-lib/src/main/java/com/microsoft/azure/toolkit/lib/containerapps/containerapp/ContainerAppDraft.java

```diff
@@ -129,12 +136,22 @@ public com.azure.resourcemanager.appcontainers.models.ContainerApp createResourc
             .withTemplate(template)
             .withWorkloadProfileName(workloadProfile)
             .create();
-        final Action<ContainerApp> updateImage = AzureActionManager.getInstance().getAction(ContainerApp.UPDATE_IMAGE).bind(this);
-        final Action<ContainerApp> browse = AzureActionManager.getInstance().getAction(ContainerApp.BROWSE).bind(this);
+        final Action<ContainerApp> updateImage = Optional.ofNullable(AzureActionManager.getInstance().getAction(ContainerApp.UPDATE_IMAGE))
+            .map(action -> action.bind(this))
+            .orElse(null);
+        final Action<ContainerApp> browse = Optional.ofNullable(AzureActionManager.getInstance().getAction(ContainerApp.BROWSE))
+            .map(action -> action.bind(this))
+            .orElse(null);
+
         AzureMessager.getMessager().success(AzureString.format("Azure Container App({0}) is successfully created.", this.getName()), browse, updateImage);
 
-        final Action<String> learnMore = AzureActionManager.getInstance().getAction(Action.OPEN_URL).withLabel("Learn More").bind("https://aka.ms/azuretools-aca-stack");
-        final Action<String> openPortal = AzureActionManager.getInstance().getAction(Action.OPEN_URL).withLabel("Open Portal").bind(this.getPortalUrl());
+        final Action<String> learnMore = Optional.ofNullable(AzureActionManager.getInstance().getAction(Action.OPEN_URL).withLabel("Learn More"))
+            .map(action -> action.bind("https://aka.ms/azure-container-apps-java"))
+            .orElse(null);
+
+        final Action<String> openPortal = Optional.ofNullable(AzureActionManager.getInstance().getAction(Action.OPEN_URL).withLabel("Open Portal"))
+            .map(action -> action.bind("https://portal.azure.com"))
```

Comment:
any reason to change the url? 
1. this action is to open the portal page of this specific resource.
2. `https://portal.azure.com` is not the right url of all Azure Environments' portal pages.

Reply:
Revert the URL. Sorry for the mistake.